lib/components/src/escapeHatch.ts

```diff
@@ -0,0 +1,7 @@
+import React, { FunctionComponent } from 'react';
+
+export function escapeHatch(component: FunctionComponent): string {
+  const name = component.displayName || component.name || 'component';
```

Comment:
If you can't find any displayName or name, I think you should return nothing. Otherwise it could lead to unexpected styling if multiple components ends up with the same classname

Reply:
We might want to help with case like:

```
<Wrapper {...props} className={`${escapeHatch(Typeset)} docblock-typeset`}>
```

If TypePreset dont provide a displayName or a name, it will return an empty string and the output will contain a leading space: `class=" docblock-typeset"`

With [classname](https://github.com/JedWatson/classnames) you can specify classes that will be returned with the conditional classes. This is pretty usefull since the consumer don't have to handle formatting /spacing  himself.

Maybe we could do something similar here so the consumer will not have to deal with spaces...

Something like:

```
export function escapeHatch(component: FunctionComponent, additionalClasses?: string): string
```

Reply:
Also, it should be renamed to something like `docsEscapeHatch` since it's specific do docs.